themes/default/content/learn/embedding-pulumi/getting-started/index.md

```diff
@@ -0,0 +1,234 @@
+---
+title: "Getting Started"
+layout: topic
+date: 2021-12-15
+draft: false
+description: |
+    Set up your local system to start building a working example of the
+    Automation API.
+meta_desc: |
+    Set up your local system to start building a working example of the
+    Automation API.
+index: 0
+estimated_time: 5
+meta_image: meta.png
+authors:
+    - laura-santamaria
+tags:
+    - learn
+block_external_search_index: false
+---
+
+Often when scaling up from an idea, an organization goes through a period of time where the engineering org is growing so fast that everyone is spinning up a lot of individual architectures in an attempt to build quickly. Eventually, the team reaches the point that someone needs to own all of the infrastructure and standard use cases have appeared. Often, once an organization understands its use cases for infrastructure, it starts to explore creating platforms for people to build on (versus everyone inventing infrastructure from scratch). And thus a platform engineering team, either only in duties or both in duties and name, is born.
+
+With all of that in mind, let's imagine that the Pulumipus Boba Tea Shop, our company from the Fundamentals learning path, has been doing so well that we can expand to jump into other markets like selling tea blends or selling its boba tea automation application to other tea shops. We're now part of a cloud engineering platform team, and we need to make it easy for developers building our applications to request the infrastructure that they need through some sort of API or perhaps a website.
```

Comment:
I would say we are now part of a platform engineering team and there are many developers from different business units building separate applications and this platform team's job is to build an interface/website/portal for people to request infrastructure